packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -630,8 +673,13 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           const { facets, state } = this;
 
           trace(`capturing oracle price `, state.updatingOracleQuote);
-          state.capturedPriceForRound = state.updatingOracleQuote;
-          void facets.helper.publishBookData();
+          if (!AmountMath.isEmpty(state.updatingOracleQuote.numerator)) {
+            state.capturedPriceForRound = state.updatingOracleQuote;
+            void facets.helper.publishBookData();
+          } else {
+            // if the price has feed has died, try restarting it.
+            facets.helper.observeQuoteNotifier();
+          }
```

Comment:
since it's exceptional, consider separating it from the normal flow.
```suggestion
          if (AmountMath.isEmpty(state.updatingOracleQuote.numerator)) {
            // if the price has feed has died, try restarting it.
            facets.helper.observeQuoteNotifier();
            return;
          }
          state.capturedPriceForRound = state.updatingOracleQuote;
          void facets.helper.publishBookData();
```

Reply:
good idea.  done.